internal/plan/plan.go

```diff
@@ -404,6 +431,53 @@ func (c *ExecCheck) Merge(other *ExecCheck) {
 	}
 }
 
+// LogTarget specifies a remote server to forward logs to.
+type LogTarget struct {
+	Name      string        `yaml:"-"`
+	Type      LogTargetType `yaml:"type"`
+	Location  string        `yaml:"location"`
+	Selection Selection     `yaml:"selection,omitempty"`
+	Override  Override      `yaml:"override,omitempty"`
+}
+
+// LogTargetType defines the protocol to use to forward logs.
+type LogTargetType string
+
+const (
+	LokiTarget   LogTargetType = "loki"
+	SyslogTarget LogTargetType = "syslog"
+	UnsetTarget  LogTargetType = ""
+)
+
+// Selection describes which services' logs will be forwarded to this target.
+type Selection string
+
+const (
+	OptOutSelection   Selection = "opt-out"
+	OptInSelection    Selection = "opt-in"
+	DisabledSelection Selection = "disabled"
+	UnsetSelection    Selection = ""
```

Comment:
This is fine to miss the log context, though. I can see ourselves using this same idea elsewhere.